### PR TITLE
Update on how to submit PR for performance repo

### PR DESCRIPTION
**Pull request by maryamariyan**
Update the docs to explain how to run benchmarks locally after corefx PR has been merged and a nuget package is available with your changes.

cc: @jorive

### CODE REVIEW

File: docs/benchmarking-workflow-corefx.md [hunk 1]
```diff
@@ -262,4 +262,11 @@ Sample project file change:
...
+The first thing you need to do is send a PR with the new API to the CoreFX repository. Once your PR gets merged and a new NuGet package is published to the CoreFX NuGet feed, you should remove the Reference to a `.dll` and install/update the package consumed by [MicroBenchmarks](../src/benchmarks/micro/MicroBenchmarks.csproj). You can do this by running the following script locally:
+
+```cmd
+py.exe scripts\benchmarks_ci.py --filter $YourFilter -f netcoreapp3.0 --incremental no --architecture x64
+```
+This script will try to pull the latest .NET Core SDK from CoreFX nightly build, which should contain the new API that you just merged in your first PR, and use that to build MicroBenchmarks project and then run the benchmarks that satisfy the filter you provided. 
+
+After you have confirmed your benchmarks successfully run locally, then your PR should be ready for performance repo.
```
Comment (ahsonkhan) [inline docs/benchmarking-workflow-corefx.md]: @jorive. @adamsitnik At what point can we merge the PR in the performance repo (i.e. when does the dependencies get updated)?

Comment (billwert) [reply docs/benchmarking-workflow-corefx.md]: right now it triggers at 1am every night. very soon it should be running any time core-sdk makes a build.

File: docs/benchmarking-workflow-corefx.md [hunk 2]
```diff
@@ -262,4 +262,11 @@ Sample project file change:
 
 Because the benchmarks are not in the CoreFX repository you must do two PR's.
 
-The first thing you need to do is send a PR with the new API to the CoreFX repository. Once your PR gets merged and a new NuGet package is published to the CoreFX NuGet feed, you should remove the Reference to a `.dll` and install/update the package consumed by [MicroBenchmarks](../src/benchmarks/micro/MicroBenchmarks.csproj). After that, you should send a PR to the performance repo.
+The first thing you need to do is send a PR with the new API to the CoreFX repository. Once your PR gets merged and a new NuGet package is published to the CoreFX NuGet feed, you should remove the Reference to a `.dll` and install/update the package consumed by [MicroBenchmarks](../src/benchmarks/micro/MicroBenchmarks.csproj). You can do this by running the following script locally:
+
+```cmd
+py.exe scripts\benchmarks_ci.py --filter $YourFilter -f netcoreapp3.0 --incremental no --architecture x64
```
Comment (jorive) [inline docs/benchmarking-workflow-corefx.md]: nit: I would remove the `py.exe` (Windows only) part from this command example, and leave it as:
```cmd
$Python3 ./scripts/benchmarks_ci.py --filter $YourFilter -f netcoreapp3.0 --incremental no --architecture x64
```

**Comment by maryamariyan** [comment]
@jorive I don't have merge privileges. Could you please merge this PR or if you approve to giving merge privileges I could merge also.
Thanks

**Comment by billwert** [comment]
I would like to glance at this first. I just haven’t had a chance yet.

Thanks,
-Bill
________________________________

**Comment by billwert** [comment]
Oh, sorry for the delay. I thought this was the other PR (#410) from my phone. This looks great.

**Comment by adamsitnik** [comment]
@maryamariyan big thanks for updating the docs!

**Pull request merged**